packages/core/src/new-api/internal/constants.ts

```diff
@@ -0,0 +1,28 @@
+import { DeployConfig } from "../types/deployer";
+
+/**
+ * The interval, in milliseconds, between checks to see if a new block
+ * has been created
+ */
+const DEFAULT_BLOCK_POLLING_INTERVAL = 200;
+
+/**
+ * The amount of time, in milliseconds, to wait on a transaction to
+ * confirm before timing out
+ */
+const DEFAULT_TRANSACTION_TIMEOUT_INTERVAL = 3 * 60 * 1000;
+
+/**
+ * The number of block confirmations to wait before considering
+ * a transaction to be confirmed during Ignition execution.
+ */
+const DEFAULT_BLOCK_CONFIRMATIONS = 0;
+
+/**
+ * Ignitions default deployment configuration values.
+ */
+export const defaultConfig: DeployConfig = {
```

Comment:
The alternative is to name the file `defaultConfig.ts`.

Reply:
sounds good